kvirt/config.py

```diff
@@ -655,13 +656,23 @@ def create_vm(self, name, profile, overrides={}, customprofile={}, k=None,
                         os.makedirs(destdir, exist_ok=True)
                     common.fetch("%s/%s" % (onfly, script), destdir)
                 script = os.path.expanduser(script)
-                if basedir != '.':
+                if basedir != '.' and not script.endswith('join.sh'):
```

Comment:
i think it s wrong to handle a script named join.sh in a different way

Reply:
Yeah I don't love this approach either. But, there's already an exception for the register.sh script ... so I figured it wouldn't be that bad 😉

Reply:
the exception for the register.sh script is to avoid duplicates from days where i didnt support natively rhnregister stuff, but i think it s actually time to remove it :)